PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitJobUploadStorageModel.kt

```diff
@@ -52,5 +53,7 @@ class SparkSubmitJobUploadStorageModel {
     @Attribute("storage_account_type")
     var storageAccountType: SparkSubmitStorageType = SparkSubmitStorageType.DEFAULT_STORAGE_ACCOUNT
 
-    @Transient var errorMsg: String? = SparkSubmissionJobUploadStoragePanel().notFinishCheckMessage
+    @get:Transient @set:Transient var errorMsg: String? = null
+
+    fun getCredentialAzureBlobAccount(): String = SERVICE_NAME_PREFIX + storageAccount.orEmpty()
```

Comment:
Should the method return `String?` and `null` when storageAccount is `null` or blank?